README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
-  <img src="./assets/logo.png" width="200" alt="Ignition Robotics" />
-  <h1>Gazebo</h1>
+  <img src="./assets/logo.png" height="200" alt="Ignition Robotics" />
```

Comment:
It's more common to set a `width`, and let `height` adjust accordingly. I'd revert this.

Reply:
I was experimenting with some other images and I forgot to revert this. Thanks for reviewing!